docs/tutorials/Vector-data-on-ground.md

```diff
@@ -105,9 +103,9 @@ var floodLayer = new itowns.ColorLayer('flood', {
 view.addLayer(floodLayer);
 ```
 
-Doing this will result visually in nothing.
-That is because we did not yet define a `Style` for our vector data. 
-To be more precise, our data consist in polygons, and we did not tell iTowns which appearance it should use to display these polygons, so iTowns did not display them.
+If you run the example now, you will notice that nothing is displayed yet.
+That is because we did not yet define a `{@link Style}` for our vector data. 
+To be more precise, our data consists in polygons, and we did not tell iTowns how these polygons should be displayed, so iTowns don't displays them.
```

Comment:
`doesn't display` instead of `don't displays` :)

Reply:
Ouch, another bad one for my english teachers :grin: